app/javascript/stylesheets/account/style.scss

```diff
@@ -0,0 +1,36 @@
+body.account {
+  .greetings {
+    color: #ffffff;
+  }
+  .header {
+    background-image: none;
+    background-color: #448ab5a3;
+    height: fit-content;
+  }
+  .container__box {
+    background-image: url("../../../public/golovna.png");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-size: cover;
+    text-shadow: 3px -1px #9c94944f;
+  }
+  .container__box > .container-fluid {
+    background: #e6e6e68f;
+    box-shadow: 0 16px 13px #e6e6e68f;
+    background-size: cover;
+    min-height: 630px;
+  }
+  .container-fluid {
+    .billing-contract {
+      h1 {
+        margin-top: 0;
+        padding-top: 30px;
+      }
+    } 
+    .bg-light {
+      background-color: #f8f9fa6e !important;
```

Comment:
try to avoid using "!important" just write more complex selector to override other value